### PR TITLE
completed card component

### PR DESCRIPTION
**Pull request by Rehmatulla-Khan**

### CODE REVIEW

File: src/components/card/contactInfoCard/ContactInfoCard.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useTheme } from "../../themeProvider/ThemeProvider";
+
+const ContactInfoCard = () => {
+  const darkMode = useTheme();
+
+  return (
+    <div className={`ui fluid link card  ${darkMode ? "inverted" : ""}`}>
+      <div className="ui basic segment">
```
Comment (faran4engg) [inline src/components/card/contactInfoCard/ContactInfoCard.jsx:15]: you need dark them for segment too ? i m not sure. Just asking

File: src/components/card/contactInfoCard/ContactInfoCard.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <div className="ui image medium centered">
+          <img
+            alt="avatar"
+            src={`https://avatars.dicebear.com/api/bottts/${
+              Math.random() * (100000 - 1) + 1
+            }.svg`}
```
Comment (faran4engg) [inline src/components/card/contactInfoCard/ContactInfoCard.jsx]: extract this logic in a util, check comment on Tanvir's PR

File: src/components/card/contactInfoCard/ContactInfoCard.jsx (hunk 1 above)
Comment (Rehmatulla-Khan) [reply src/components/card/contactInfoCard/ContactInfoCard.jsx:15]: no its inheriting @faran4engg

File: src/components/card/contactInfoCard/ContactInfoCard.jsx (hunk 2 above)
Comment (Rehmatulla-Khan) [reply src/components/card/contactInfoCard/ContactInfoCard.jsx]: okay @faran4engg

File: src/pages/home/Home.jsx [hunk 3]
```diff
@@ -12,28 +13,41 @@ const Home = () => {
...
+          <div className="item">
+            <i className="sign out alternate icon" onClick={toggleTheme}></i>
           </div>
         </div>
       </nav>
+      <main style={{ marginTop: "6rem" }}>
+        <div className="container">
+          <div className="ui four column doubling stackable grid container">
+            {[...new Array(20)].map(() => (
+              <div className="column">
```
Comment (faran4engg) [inline src/pages/home/Home.jsx]: chaabi kaha h ?
chabi

Comment (Rehmatulla-Khan) [reply src/pages/home/Home.jsx]: haa bhai daalta hu 😂

**Pull request merged**